apis/clouddeploy/v1alpha1/deliverypipeline_types.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	commonv1alpha1 "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/common/v1alpha1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/k8s/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var DeployDeliveryPipelineGVK = GroupVersion.WithKind("DeployDeliveryPipeline")
```

Comment:
What did you change about the "Deploy" and "CLouDeploy"?  Is there another service called "Deploy"? If so, I like the directory change you made and I think this should then be `ClouDeployeDeliveryPipeline` as well. Otherwise, we may want to keep the directory as `apis/deploy/v1alpha1/...` and this Kind is correct.

Reply:
Discussed offline, we're going to rename the kind to CloudDeployXYZ